tests/test_pysyncrosim.py

```
@@ -360,14 +360,91 @@ def test_library_delete():
         
     with pytest.raises(ValueError, match="scenario dne does not exist"):
         myLibrary.delete(scenario="dne")
+    
+    with pytest.raises(TypeError, match="folder must be a Folder instance or Integer"):
+        myLibrary.delete(folder="folder")
+
+    with pytest.raises(ValueError, match="Folder ID 50 does not exist"):
+        myLibrary.delete(folder=50, force=True)
+
+    
+    myLibrary.delete(folder=myFolder, force=True)
```

Ruff (0.14.11)

Undefined name `myFolder`

(F821)